apps/dashboard/lib/services/user-import.service.ts

```
@@ -0,0 +1,60 @@
+      const response = await axios.post(`${SERVER_API_URL}/user/commit-imported-users`, {
+        users,
+        organizationId,
+      });
+
+      return response.data;
+    } catch (error) {
+      console.error('Failed to import users:', error);
+      throw error;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)